### PR TITLE
[Issue #75] Added sticky behavior to search filter sidebar

### LINKED ISSUE #75: [Task]: Make search sidebar sticky
### Summary

Make the search sidebar use the sticky class so that it can be scrolled independently of the results when stuck.

See https://codepen.io/andycochran/pen/BabejWq?editors=1100

This is blocked by [ issue ??? ] to implement the beta alert banner, which also uses sticky. It needs to be in place first to make sure these can both work together.

### Acceptance criteria

- Search sidebar Is sticky

### PR DESCRIPTION
**Pull request by btabaska**
## Summary
Fixes #75

### CODE REVIEW

File: frontend/src/app/[locale]/search/SearchForm.tsx [hunk 1]
```diff
@@ -60,26 +60,28 @@ export function SearchForm({
...
-            />
-            <SearchFilterAgency
-              formRef={formRef}
-              initialQueryParams={agencyQueryParams}
-            />
-            <SearchFilterCategory
-              formRef={formRef}
-              initialQueryParams={categoryQueryParams}
-            />
+            <div className="position-sticky top-0 maxh-viewport overflow-auto">
```
Comment (rylew1) [inline frontend/src/app/[locale]/search/SearchForm.tsx]: nit: maybe pull these classes into their own custom style `className="searchFormStickyBar"`

**Comment by rylew1** [comment]
I'll let @andycochran take a peek but looks good!

File: frontend/src/app/[locale]/search/SearchForm.tsx (hunk 1 above)
Comment (acouch) [reply frontend/src/app/[locale]/search/SearchForm.tsx]: We should have a standard and stick to it (either prefer custom class names or not) and document. I don't have a strong opinion either way.

Comment (btabaska) [reply frontend/src/app/[locale]/search/SearchForm.tsx]: Iterated based on that feedback.

Comment (btabaska) [reply frontend/src/app/[locale]/search/SearchForm.tsx]: My opinion is that USWDS already supports custom CSS in the _uswds-theme-custom-styles file and we have already leveraged that for the .pdf-card component. It is cleaner to move large chunks of CSS into a single file to make it easier to read inline.

Comment (andycochran) [reply frontend/src/app/[locale]/search/SearchForm.tsx]: If we add custom classes, they shouldn't duplicate the function of utility classes. USWDS already has a class that makes something sticky, no need to define a new style to do the same thing. I think custom classes should only be used for styles that are not possible with utility classes.

**Comment by andycochran** [comment]
We might hold off on this ticket. Honestly, it was sort of a placeholder/note-to-self when I wrote it. I imagined it might work a little differently than it's implemented here. It's a bit awkward that it has a max-height and scrolls independently while not stuck. Since there's no `:is-stuck` CSS selector, it'd be quite a bit of work to detect that state ([e.g. link](https://css-tricks.com/how-to-detect-when-a-sticky-element-gets-pinned/)) and only apply the max-height then. Again, maybe we hold off on this. I'm not sure it's necessary, and the filter design is likely to change quite a lot. 

The downside to not doing this is that the scroll position of the results and filter are intertwined. And we've not done any usability testing that shows this is a problem. It's an assumption that users will want to retain the results scroll position while scrolling through the filters.

**Pull request closed without merging**